src/components/Charts/ChartRt.tsx

```diff
@@ -0,0 +1,256 @@
+import React from 'react';
+import { isUndefined, tail as _tail } from 'lodash';
+import { min as d3min, max as d3max } from 'd3-array';
+import { Group } from '@vx/group';
+import { ParentSize } from '@vx/responsive';
+import { scaleLinear, scaleTime } from '@vx/scale';
+import { GridRows } from '@vx/grid';
+import { curveNatural } from '@vx/curve';
+import { AxisBottom, AxisLeft } from '@vx/axis';
+import { LinePath, Area } from '@vx/shape';
+import { RectClipPath } from '@vx/clip-path';
+import { useTooltip } from '@vx/tooltip';
+import { localPoint } from '@vx/event';
+import { ProjectionDataset, RT_TRUNCATION_DAYS } from '../../models/Projection';
+import { CHART_END_DATE, CASE_GROWTH_RATE, Zones } from '../../enums/zones';
+import BoxedAnnotation from './BoxedAnnotation';
+import HoverOverlay from './HoverOverlay';
+import {
+  formatDecimal,
+  getChartRegions,
+  getTruncationDate,
+  getZoneByValue,
+  last,
+  randomizeId,
+} from './utils';
+import * as Style from './Charts.style';
+
+const computeTickPositions = (minY: number, maxY: number, zones: Zones) => {
+  const maxZones = zones.MEDIUM.upperLimit;
+  const maxTick = maxY < maxZones ? 1.5 * maxZones : maxY;
+  return [minY, zones.LOW.upperLimit, zones.MEDIUM.upperLimit, maxTick];
+};
+
+// Accessing the data
+const x = (d: any) => new Date(d.x);
+const yRt = (d: any) => d?.y?.rt;
+const yHigh = (d: any) => d?.y?.high;
+const yLow = (d: any) => d?.y?.low;
```

Comment:
**TODO**: Since we know the type of `projectionDataset`, we _should_ be able to get rid of all the `any` here, not sure what the correct type would be though.

Reply:
yeah would be good preferrable to make theset type safe